pygsl/meson.build

```diff
@@ -0,0 +1,14 @@
+#: Todo: revisit if there is a more elegant way for this
+py_sources = run_command('find',
+['.',  '-maxdepth', '1', '-name', '*.py'],
+  check : true
+).stdout().strip().split('\n')
+
+py.install_sources(
+  py_sources,
+  pure: false,
```

Comment:
`pure: false` is implicit from:
```meson
py = import('python').find_installation(pure: false)
```

If you set it in find_installation, you set the defaulted value for all calls to `py.xxx()` methods which have a "pure" kwarg.